conjure-go-client/httpclient/config.go

```diff
@@ -132,85 +132,89 @@ func (c ServicesConfig) ClientConfig(serviceName string) ClientConfig {
 	}
 	conf.ServiceName = serviceName
 
+	return MergeClientConfig(conf, c.Default)
+}
+
+func MergeClientConfig(conf, defaults ClientConfig) ClientConfig {
```

Comment:
Mind adding a short godoc comment here? It may be worth mentioning the "service-name" field is expected to be set as it's not touched here, although I would assume most users retrieve the client config via the methods above, which do set the service-name.

Reply:
Updated!